### PR TITLE
[no ticket] bump graal

### PR DESCRIPTION
**Pull request by Qi77Qi**
\<your comments for this PR go here\>

---
Have you read [CONTRIBUTING.md](https://github.com/DataBiosphere/leonardo/blob/develop/CONTRIBUTING.md) lately? If not, do that first.

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; Delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rtitle — APPROVED** [review]

File: docker/build.sh [hunk 1]
```diff
@@ -175,8 +175,9 @@ function docker_cmd()
 
         docker build -t "${DEFAULT_IMAGE}:${DOCKER_TAG}" .
 
-        echo "scanning docker image..."
-        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "${DEFAULT_IMAGE}":"${DOCKER_TAG}"
+# TODO: enable trivy again https://broadworkbench.atlassian.net/browse/IA-2291
+#        echo "scanning docker image..."
+#        docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v "$HOME"/Library/Caches:/root/.cache/ aquasec/trivy --exit-code 1 --severity CRITICAL "${DEFAULT_IMAGE}":"${DOCKER_TAG}"
```
Comment (rtitle) [inline docker/build.sh:180]: Maybe we could still run it without `--exit-code 1` to report the errors but not fail the build?

Comment (rtitle) [reply docker/build.sh:180]: Also did it in my PR: https://github.com/DataBiosphere/leonardo/pull/1652

Comment (Qi77Qi) [reply docker/build.sh:180]: oh I see..

> Maybe we could still run it without --exit-code 1 to report the errors but not fail the build?
Good idea!

I'll wait for your PR then if you've already made the change :)

**Comment by Qi77Qi** [comment]
duplilcate of https://github.com/DataBiosphere/leonardo/pull/1652

**Pull request closed without merging**